openapi/components/schemas/CreditMemos/CreditMemoProperties.yaml

```
@@ -0,0 +1,176 @@
+id:
+  description: Credit memo ID.
+  readOnly: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+invoiceId:
+  description: ID of the invoice which the credit memo is issued upon.
```

nitpick: suggestion to avoid ending in a preposition
 
```suggestion
  description: ID of the invoice on which the credit memo is issued.
```